src/libstd/os/mod.rs

```diff
@@ -24,7 +24,7 @@ cfg_if::cfg_if! {
         // If we're not documenting libstd then we just expose the main modules
         // as we otherwise would.
 
-        #[cfg(any(target_os = "redox", unix))]
+        #[cfg(any(target_os = "redox", unix, target_os = "vxworks"))]
```

Comment:
To confirm, is vxworks unix-like-enough that you'll want `std::os::unix` to be a module? Or is it different enough that modules should use something like `std::os::vxworks`?

Reply:
Honestly, I am not sure which way is better, I think it is unix-like-enough.